go.sum

```diff
@@ -0,0 +1,26 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I think this will cause circular imports? Do we need to split the cloud repo first...?

Reply:
Why does genproto need cloud.google.com/go?

Reply:
Yikes. Yeah. I'll go make the relevant mod files. The path is:

genproto -> grpc -> oauth2 -> cloud.google.com/go/compute/metadata

Reply:
I've come to the conclusion that must be allowed if we want to add a go.mod file in the near future:

- Cyclic module dependencies are OK [1]. Cyclic _package_ dependencies are not OK and enforced by the compiler.
- x/oauth2/google must get a mod file that specifically requires `cloud.google.com/go/compute/metadata` instead of `cloud.google.com/go` to resolve this circular dependency.
- x/ repos are quite a ways out from getting modules [2] [3].

We could wait for the x/ changes, but I think given the first point it's OK to go ahead now. WDYT?

1: https://groups.google.com/forum/#!searchin/golang-nuts/import$20cycle%7Csort:date/golang-nuts/h2r8ktgOOZI/Pm6uyj5BCQAJ
2: https://github.com/golang/go/issues/28136
3: https://github.com/golang/go/issues/27858

Reply:
This is slightly unfortunate but sounds fine with me.